Level2/Echai_funding/README.md

```diff
@@ -42,12 +42,17 @@ https://drive.google.com/drive/folders/1fM9g84aN4GYZfJp8QN-ps5xertshxx-V?usp=sha
 
 
 ## HOW TO RUN THIS PROJECT ON YOUR PC:
- steo 1: clone the repo
- step 2: install hardhat and react according to dosumentation.
+ step 1: clone the repo.
+ 
+ step 2: install hardhat (link =https://hardhat.org/hardhat-runner/docs/getting-started#overview) and react (link= https://reactjs.org/docs/getting-started.html) package according to documentation.
+ 
  step 3: setup your own metamask walleet address and alchemy api key via .env file.
- step: test smart contract with already written test cases.
- step4: if you face any issue feel free to dm me GitHub = @DevSwayam
  
+ step4: Deploy and test smart contract with already written test cases.
+ 
```

Comment:
after step4 write 
do `npm install` to install all dependencies

Reply:
Bro I already told to reffer hardhat and react documentation for installation it's already included in it

Reply:
'npm install' is used to dowoad the dependencies that is being used by your project . if someone doesn't know about all this then by doing this all drpendecies will installed . hope u get this .

Reply:
Check I have included